docs/api/assets/site.js

```diff
@@ -39,6 +39,7 @@ function showHashTarget(hash) {
   var targetId = hash && hash.substring(1);
   if (!targetId) return;
   var hashTarget = document.getElementById(targetId);
+  if (!hashTarget) return;
```

Comment:
This change seems to have been introduced by a merge commit. Per [`CONTRIBUTING.md`](https://github.com/mapbox/mapbox-gl-js/blob/master/CONTRIBUTING.md#git-conventions) we require rebases, not merge commits, in this repo. Let me know if you need any help doctoring the git history.